frontend/components/Home.jsx

```diff
@@ -22,28 +22,26 @@ import "@plasmicapp/react-web/lib/plasmic.css";
 import "./plasmic/plasmic__default_style.css"; // plasmic-import: global/defaultcss
 import "./plasmic/canvas_app_explorer/plasmic_canvas_app_explorer.css"; // plasmic-import: mXv5TZ5SUPGRneH9RoMn6q/projectcss
 import "./plasmic/canvas_app_explorer/PlasmicHome.css"; // plasmic-import: 4XPgsAhljqdds/css
-
 export const Home__VariantProps = new Array();
-
 export const Home__ArgProps = new Array();
-
 function Home__RenderFunc(props,ref) {  
+    const [addedTools, setAddedTools] = useState([]); // each tool has one entry in array
+    const [learnMoreActive, setLearnMoreActive] = useState([]) // each tool has one entry in array
     const [tools, setTools] = useState(null);
 
     useEffect(async () => {
         const url = "http://localhost:5000/api/lti_tools/";
         const response = await fetch(url);
         const data = await response.json();
         setTools(data)
+        setAddedTools(Array(Object.keys(data).length + 2).fill(false))
```

Comment:
I don't think we really need to initialize this and can just leave it undefined and set the values that are enabled to true. This may also change to something like an object instead of an array.

Ultimately we'll be using the [tabs API in Canvas](https://canvas.instructure.com/doc/api/tabs.html) to get the tools that are placed and map them to the tools in our system.

That API returns a tool like
```
  {
    "id": "context_external_tool_12345",
    "html_url": "/courses/123456/external_tools/12345",
    "full_url": "https://umich.instructure.com/courses/123456/external_tools/12345",
    "position": 18,
    "visibility": "public",
    "hidden": true,
    "label": "Zoom",
    "type": "external",
    "url": "https://umich.instructure.com/api/v1/courses/123456/external_tools/sessionless_launch?id=12345&launch_type=course_navigation"
  },
```

So maybe we'll have something here like

toolsHidden: {
  "12345": false,
  "23456": true
}

Probably something like this idea to get it to update.
https://stackoverflow.com/a/54150873/3708872

It's fine to push that off to a later task.